exp/kubeadm-ignition/api/v1alpha2/conversion.go

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
We can probably remove the v1alpha2 APIs completely

Reply:
> We can probably remove the v1alpha2 APIs completely

Removed v1alpha2 completely.